src/components/Common/UI/Calendar/Calendar.module.scss

```diff
@@ -0,0 +1,258 @@
+:global(.GSDK) {
```

Comment:
I accidentally deleted this stylesheet when cleaning up date picker. This restores it. It implements the same styles found here https://github.com/Gusto/embedded-react-sdk/pull/184